Src/AutoNSubstituteUnitTest/NSubstituteRegisterCallHandlerCommandTest.cs

```diff
@@ -10,7 +10,7 @@
 
 namespace AutoFixture.AutoNSubstitute.UnitTest
 {
-    public class NSubstituteRegisterCallHandlerCommandTest
+    public partial class NSubstituteRegisterCallHandlerCommandTest
```

Comment:
You *really* need `partial` classes?

Reply:
Well, it might be rewritten without `partial`, but here it's very convenient usage. It's used only for compatibility tests, so to me it suits perfectly 😊 

C'mon, I learned about that feature from the very first day and all this time I waited for the chance to finally use it 😅  Let me do it at least once in my life 😊

Reply:
I mean, yeah, go for it, but I don't think it's reasonable as he or she who reads the code must remember to jump into the other part(s) of the partial type definition.